fem/integ/bilininteg_elasticity_kernels.hpp

```diff
@@ -164,8 +180,8 @@ void ElasticityAddMultPA_(const int nDofs, const FiniteElementSpace &fespace,
                                                  ir);
    E_To_Q_Map->SetOutputLayout(QVectorLayout::byNODES);
    // interpolate physical derivatives to quadrature points.
-   E_To_Q_Map->PhysDerivatives(x, QVec);
-
+   if (!useAbs) { E_To_Q_Map->PhysDerivatives(x, QVec); }
+   else { E_To_Q_Map->AbsPhysDerivatives(x, QVec); }
```

Comment:
To avoid the proliferation of `*Abs*` methods to `QuadratureInterpolator`, instead of using `AbsPhysDerivatives` here, this absolute value should be done inside the kernel below. This way, no changes are needed in `QuadratureInterpolator`.